mava/systems/q_learning/types.py

```diff
@@ -32,7 +32,9 @@ class Transition(NamedTuple):
     obs: Observation
     action: Array
     reward: Array
-    done: Array
+    terminal: Array
+    term_or_trunc: Array
+    next_obs: Observation
```

Comment:
Would be nice to have an explanation here also
```suggestion
    # Even though we use a trajectory buffer we need to store both obs and next_obs.
    # This is because of how the `AutoResetWrapper` returns obs at the end of an episode.
    next_obs: Observation
```